### PR TITLE
1er jet avec new user et login et rules page

### PR DESCRIPTION
**Pull request by serhanif59**

### CODE REVIEW

**Review by n3en — APPROVED** [review]

File: src/Form/UserType.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class UserType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('firstname', null, ['label' => false, 'attr' => ['placeholder' => 'Prénom']])
+            ->add('lastname', null, ['label' => false, 'attr' => ['placeholder' => 'Nom']])
+            ->add('email', null, ['label' => false, 'attr' => ['placeholder' => 'Email']])
+            ->add('nickname', null, ['label' => false, 'attr' => ['placeholder' => 'Pseudo']])
+            ->add('password', PasswordType::class, ['label' => false, 'attr' => ['placeholder' => 'Mot de passe']])
```
Comment (Benja-san) [inline src/Form/UserType.php:21]: Optional: Add a password confirmation ?

File: templates/base.html.twig [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
-        {# Run `composer require symfony/webpack-encore-bundle` to start using Symfony UX #}
-        {% block stylesheets %}
-            {{ encore_entry_link_tags('app') }}
-        {% endblock %}
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<link rel="icon" type="image/png" href="{{ asset('build/images/logo4.png') }}"/>
+		<title>
+			{% block title %}H&G
```
Comment (Benja-san) [inline templates/base.html.twig]: Optional : If H&G is always present in all pages, you can put it just above the title block, by this process you won't have to put the {{ parent() }} twig function in every title block, but ust the " - "pagename"

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-        </h1>
-    </div>
-    <div class="white-stripe">
-        <img src="{{ asset('build/images/LOGO_WILD_CODE_SCHOOL.png') }}"
-             alt="">
-    </div>
+{% block content %}
+	<div class="pink-stripe">
+		<h1>
+			Let's start coding&nbsp;&#129304;
```
Comment (Benja-san) [inline templates/home/index.html.twig:9]: Let's start coding ?

File: templates/layout.html.twig [hunk 4]
```diff
@@ -0,0 +1,40 @@
+<!doctype html>
```
Comment (Benja-san) [inline templates/layout.html.twig]: You can Delete this file

File: templates/user/_addUserForm.html.twig [hunk 5]
```diff
@@ -0,0 +1,36 @@
+{% block content %}
```
Comment (Benja-san) [inline templates/user/_addUserForm.html.twig:1]: Delete All useless files

File: templates/user/_editUserForm.html.twig [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{% block content %}
```
Comment (Benja-san) [inline templates/user/_editUserForm.html.twig:1]: Delete or gitignore this one too

File: templates/user/editUser.html.twig [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html.twig' %}
```
Comment (Benja-san) [inline templates/user/editUser.html.twig]: giignore ro delete this file

File: templates/user/index.origin.html.twig [hunk 8]
```diff
@@ -0,0 +1,73 @@
+{% extends 'base.html.twig' %}
```
Comment (Benja-san) [inline templates/user/index.origin.html.twig]: gitignore or delete this file

File: templates/user/show.origin.html.twig [hunk 9]
```diff
@@ -0,0 +1,91 @@
+{% extends 'base.html.twig' %}
```
Comment (Benja-san) [inline templates/user/show.origin.html.twig]: delete or gitignore this file

**Review by Benja-san — APPROVED** [review]
Approved, please remove all useless files and change title block before merging

**Pull request merged**